pages/consortium.html

```diff
@@ -0,0 +1,140 @@
+<#def TITLE>The Coq Consortium</#def>
+<#include "incl/header.html">
+
+<div class="framework">
+<div class="frameworklabel">A pool of resources and a platform for exchanges</div>
+<div class="frameworkcontent">
+
+<p>The objective of the Consortium is to make the community of users feel empowered with respect to
+    the future of the Coq system, giving them opportunities to see how the system will evolve, where it is
+    going, and to take an active part in this process. On the one hand, we wish to establish <strong>transparent
+    ways of contributing financially and technically</strong> to the development of the system; on the other
+    hand we wish to make sure that the <strong>most pressing needs of the user community are addressed
+    efficiently</strong> by providing a fantastic working environment to some of the most competent developers
+    trained in our teams.</p>
```

Comment:
Although this is not explicit, I have this impression is to centered on inria based teams.  I propose changing to
"to competent developers"

Reply:
I'm not sure I understand exactly the proposal. Is it only to remove "most"? Or also "trained in our teams"?

Reply:
remove "trained in our teams"